### DISCUSSION TITLE
A way to get ID back without hiring then slaughtering crew

### DISCUSSION THREAD

**Discussion by DougStep** [FakeFishGames/Barotrauma#7184] [Ideas and suggestions]
AFAIK in solo at least, and from what I've found in discussions online should you lose an ID card the only way to get a new one is to hire then kill crew. 
Say you have to abandon a case you're carrying so you don't die but your ID was in it while you were using a wreck ID
or you swap your ID out in a wreck and don't notice and wind up without access to your own ship you should be able to somehow get a new ID card WITHOUT mericless slaughter.

Suggestions for methods: 
1. Crafted at assembler   
2. Purchased at a station store
3. Combining both, Craft the card at an assembler, then have it verified and activated at a station store
4. Buying fake ID that is funcional but maybe has a debuff untill you can find someone that can create a new propper ID for you   

Edit: Bonus afterthought, if it's a concern everyone grabbing captains ID cards simply have creating a new ID for the same character invalidates the other card

**Comment by trevormoney1** [comment]
Entirely different idea: Allow for server moderators specifically to be able to craft ID cards from a single plastic, and allow for the sub editor to have an ID registry, including all ID tags listed at spawns as well as any other IDs the submarine creator wants to add. This would allow for sub editors to have more detailed authorization mechanics. Crafting an ID will allow the moderator to select permissions to add to the ID and give to the player. A player name could be selected from the player list to give to the ID, or an entirely new ID system could be made, giving them an inspectable element similar to reactor PDAS and the like where you can view information about the ID and the name of the person.

**Comment by James-E-A** [comment]
> Craft the card at an assembler, then have it verified and activated at a station store

That sounds like a fantastic idea -- actually, why not let the captain manage _all_ ID cards via the station staff NPC? Being able to reprogram ID cards for 5mk to add/remove additional tags (`access_armory`, `access_command`, `access_quarters`, `access_ballast`, `access_medical`, etc.) could allow a lot more collaboration between captains and sub editors, too.

Just like IRL access control systems, you should be able to use the management interface to _deactivate_ any recognized cards that you suspect have been lost, or stolen or maybe even duped by traitors. ("Deactivated" cards still work for disguise -- only that the _sub_ won't let them into anywhere.)

Recipe should probably be 1 plastic + 1 FPGA.

**Reply by James-E-A** [reply]
Re stealing/duping cards: should storage containers (or, at the very least, crates) work as a faraday cage and prevent ID cards from showing up on the status monitor?
